Add 0.26 Release Notes (#83)
bpftrace/website: src/pages/release-026-notes.md

# 0.26 Release Notes

Below are some highlights from this release. For a full list of changes, check
out the release notes on GitHub:
- [**0.26.0**](https://github.com/bpftrace/bpftrace/releases/tag/v0.26.0)

> Note that this release comes out of the traditional [release

Reviewer: nit:
```suggestion
Note that this release does not adhere to the traditional [release
```

schedule](https://github.com/bpftrace/bpftrace/blob/master/docs/release_process.md#release-cadence),
the reason mostly being that it features several important bug fixes, which
would be hard to backport to the previous release branch. Despite being
primarily about fixes, there is a number of new and exciting features.

## What's New

Two features that have been introduced in the previous releases are now finally
coming out of the "unstable" status: **imports** and the **address-of

> operator**. Next, there are several new features that leverage the DWARF

Reviewer: ```suggestion
coming out of the "unstable" status: **imports** and the **address-of
operator**; both are critical parts of [C-interop](https://bpftrace.org/blog/c-interop). Next, there are several new features that leverage the DWARF
```

debugging information, such as **stack unwinding** (unstable at the moment) and
**statement probes**. The below sub-sections describe the most significant
features in detail.

In addition to new features, there is a number of internal changes, some of

> which surface to users. The most notable one is a new *recursive-descent

Reviewer: ```suggestion
which are surfaced to users. The most notable one is a new *recursive-descent
```

parser*, which replaced the previous, more traditional, LR parser. The new

> parser allowed to define a clearer [structure of bpftrace

Reviewer: ```suggestion
parser allows us to define a clearer [structure of bpftrace
```
